sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -347,6 +353,63 @@ def create_or_update(self, data: Data) -> Data:
                     )
             raise ex
 
+    @experimental
+    def import_data(self, data_import: DataImport) -> Job:
```

Comment:
The SDK syntax for this function is below, which feels redundant. We should consider to rename this function to be `import()`
```python
mlclient.data.import_data()
```